test/controllers/tasks/psc.statement.controller.unit.ts

```diff
@@ -42,4 +44,15 @@ describe("PSC Statement controller tests", () => {
       spyGetUrlToPath.mockRestore();
     });
   });
+
+  describe("post tests", function () {
+    it("Should redisplay psc page with error when radio button is not selected", async () => {
+      const response = await request(app).post(PSC_STATEMENT_URL);
+
+      expect(response.status).toEqual(200);
```

Comment:
can you add a test to check the error gets thrown similar to what was done in the get function